cinema/views.py

```diff
@@ -58,16 +63,44 @@ class CinemaHallViewSet(
     permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
 
 
-class MovieViewSet(
-    ReadOnlyModelViewSet,
-    mixins.CreateModelMixin,
-    GenericViewSet,
-):
+class MovieViewSet(ModelViewSet):
```

Comment:
The `MovieViewSet` should use `ModelViewSet` instead of inheriting from both `ModelViewSet` and `GenericViewSet` with `CreateModelMixin`. This will simplify the viewset and ensure proper handling of both read and write operations.